### DISCUSSION TITLE
Pull request 빌드 오류

### DISCUSSION THREAD

**Discussion by yoonhyeokCho** [TEAM-DAME/Dame-Android#134] [2. Q&A]
![image](https://user-images.githubusercontent.com/90389363/205135852-555de370-c931-4910-ba11-1f58002f972b.png)

### 구글링 결과 JDK 버전 문제라던데 혹시 아시는 분이 있을까요 ?

**Comment by onseok** [comment]
buildSrc 모듈의 버전이나, 프로젝트, 모듈별 build.gradle.kts 파일들, settings.gradle 파일에 수정사항이 있는지 확인 부탁드립니다.
